components/HeaderComponent.js

```diff
@@ -88,6 +89,7 @@ function Header({ siteTitle }) {
     <header style={{ position: 'relative' }}>
       <SkipToContent />
       <ApplicationStateBanner deploymentType={deploymentType} />
+      <TopToolBar />
```

Comment:
You can use the `useSession` hook here and conditionally render this if `session.status === 'authenticated'`